app/views/admin/check_out_checkpoint_units/_form.html.slim

```diff
@@ -7,7 +7,7 @@
 = simple_form_for checkpoint_unit, url: form_url do |f|
   = f.input :check_out_ok
   = f.input :notes_check_out
-  = f.input :cost_in_chf
+  = f.input :cost_in_chf, input_html: { data: { controller: 'numericality', action: 'change->numericality#clearInvalidValues blur->numericality#clearInvalidValues' } }
```

Comment:
problem: if you type into a input[number] "10.-" it will keep that value in the field but send "".